src/Security/samples/CustomAuthorizationFailureResponse/Startup.cs

```diff
@@ -34,6 +34,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAuthorization(options => options.AddPolicy(SamplePolicyNames.CustomPolicyWithCustomForbiddenMessage, policy => policy.AddRequirements(new SampleWithCustomMessageRequirement())));
```

Comment:
While you are here can you clean this up too?  Its really weird that this is calling AddAuthorization twice, instead AddPolicy should just be called twice in one of these AddAuthorization calls

Reply:
Thanks for the note! It was really weird. Fixed.